packages/dicomImageLoader/src/types/MetadataModules.ts

```diff
@@ -0,0 +1,81 @@
+export type MetaDataTypes =
+  | 'generalSeriesModule'
+  | 'patientStudyModule'
+  | 'imagePlaneModule'
+  | 'imagePixelModule'
+  | 'transferSyntax'
+  | 'sopCommonModule'
+  | string;
+
+export interface DicomDateObject {
```

Comment:
If this is getting retrieved/parsed, we should always get it as a combined date/time object.  Splitting it up causes all sorts of issues.  If we are going to not parse, then just leaving the value as a string is preferred.